metricbeat/module/kubernetes/state_replicaset/data.go

```diff
@@ -0,0 +1,59 @@
+package state_replicaset
```

Comment:
I wanted to keep the same format for all `kube-state-metrics` metricsets for one reason, the `hosts` setting is incompatible between `kubelet` and `kube-state-metrics`, you have to choose, and it's nice to have that clear from the metricset name, to avoid confusion.

Check https://github.com/exekias/beats/blob/d091ce76089c44e5fd872661d529932aa92633df/metricbeat/module/kubernetes/_meta/config.yml for an example of possible configurations